src/components/miscellaneous/AboutUs.js

```diff
@@ -0,0 +1,68 @@
+import React, { useRef, useState, useEffect } from "react"
+import { Form, Button, Card } from "react-bootstrap"
+import { useHistory } from "react-router-dom"
+import "react-datepicker/dist/react-datepicker.css";
+import { useAuth } from "../../contexts/AuthContext"
+import Typography from "@material-ui/core/Typography";
+import { ExternalLinkIcon } from '@chakra-ui/icons'
+
+import {
+    Alert,
+    AlertIcon,
+    AlertTitle,
+    AlertDescription,
+    Box,
+    Input,
+    FormControl,
+    FormLabel,
+    FormErrorMessage,
+    FormHelperText,
+    Select,
+    Text,
+    Image,
+    Link
+} from '@chakra-ui/react';
+
+import NavigationBar from '../common/NavigationBar';
+
+import GOOSE_IMAGE from "../../images/Goose.png"
+
+import "../../styles/form-style.css";
+
+const { REACT_APP_API_ENDPOINT } = process.env;
+
+const CreateResource = () => {
+
+    return (
+        <div>
+            <NavigationBar></NavigationBar>
+
+            <Box sx={{ position: 'absolute', top: 100, left: '50%', transform: 'translateX(-50%)' }}>
+                <Card style={{ padding: '16px' }}>
```

Comment:
looks like the links are still not being centered after a restart. Here was the fix that worked for me. Replace line 41 with this:
```
<Card style={{ padding: '16px', textAlign: 'center' }}>
```

and remove all `align="center"` props from all child components. lmk if this works.

Reply:
works on my end, how about you?

Reply:
working